hvp.php

```diff
@@ -215,13 +344,28 @@ private function pathsToCsv($libraryData, $key) {
     return '';
   }
 
+  // Magic stuff not used
+  public function lockDependencyStorage() {
+
+  }
+
+  public function unlockDependencyStorage() {
+
+  }
+
   /**
-   * Implements deleteLibraryDependencies
+   * Implements deleteLibrary
    */
-  public function deleteLibraryDependencies($libraryId) {
+  public function deleteLibrary($library) {
```

Comment:
Do not mind about these parts where it seems like function name has changed. I reordered the functions to better match the order in Wordpress and Drupal plugins, and the diff doesn't understand the change.